src/common/options/global.yaml.in

```diff
@@ -790,6 +790,19 @@ options:
   level: advanced
   desc: Set the maximum number of session within Qatzip when using QAT compressor
   default: 256
+- name: uadk_compressor_enabled
```

Comment:
I'd move this down so that it's placed after `qat_compressor_busy_polling` to avoid UADK-related options being defined in the middle of QAT-related options (namely between `qat_compressor_session_max_number` and `qat_compressor_busy_polling`).

Reply:
adjusted orders